core/trino-main/src/main/java/io/trino/metadata/Metadata.java

```diff
@@ -290,6 +291,16 @@
      */
     Optional<ConnectorOutputMetadata> finishInsert(Session session, InsertTableHandle tableHandle, Collection<Slice> fragments, Collection<ComputedStatistics> computedStatistics);
 
+    /**
+     * Returns true if materialized view refresh should be delegated to connector
+     */
+    boolean isDelegateMaterializedViewRefreshToConnector(Session session, QualifiedObjectName viewName);
```

Comment:
I'd rename the method to `delegateMaterializedViewRefreshToConnector`. Remove the `is` prefix.